### PR TITLE
added reaction trails

### PR DESCRIPTION
**Pull request by jason-liau**
when a user sends a message, the bot will react with the emoji specified in config.jason

### CODE REVIEW

File: toes/triggers.py [hunk 1]
```diff
@@ -134,6 +141,26 @@ def trigger_reaction_custom(*, keyword: str, probability: float = 1.0, emoji_id:
     async def trigger(bot: Client, message: Message): ...
     return trigger
 
+@jason_trigger('reaction_trail_standard')
+def trigger_reaction_standard(*, author_id: str, probability: float = 1.0, emoji: str,  **kwargs: Any) -> Trigger:
```
Comment (96-LB) [inline toes/triggers.py:145]: We've got a naming collision with the new factory methods you added. Could you change the names to differentiate them?

File: toes/triggers.py [hunk 2]
```diff
@@ -43,6 +43,13 @@ async def if_keyword(bot: Client, message: Message, trigger: Trigger, keyword: s
     if keyword in message.content:
         await trigger(bot, message)
 
+@modifier
+async def if_author(bot: Client, message: Message, trigger: Trigger, author_id: str) -> None:
```
Comment (96-LB) [inline toes/triggers.py:47]: Any particular reason you chose a string? It'd probably be best to keep it as an integer to match with the other ID fields.

**Review by 96-LB — COMMENTED** [review]
Thanks for the PR! Left a couple comments for you to take a look at.

File: toes/triggers.py (hunk 1 above)
Comment (jason-liau) [reply toes/triggers.py]: fixed

File: toes/triggers.py (hunk 2 above)
Comment (jason-liau) [reply toes/triggers.py]: fixed

**Comment by 96-LB** [comment]
Looks good.

**Pull request merged**